mysql-test/std_data/keyring_vault_confs/keyring_vault.conf

```diff
@@ -0,0 +1,4 @@
+vault_url = https://127.0.0.1:8200
```

Comment:
I have re-factored tests and now should be possible to run all keyring_vault tests in parallel - thus they can be run as normal tests on Jenkins. I have requested a server with HashiCorp Vault on it - that the tests running on Jenkins will connect to.